spec/features/saml/loa3_sso_spec.rb

```diff
@@ -66,14 +75,38 @@ def sign_out_user
 
   context 'canceling verification' do
     context 'with js', js: true do
+      let(:warning_qualifier) { t('idv.cancel.warning_qualifier') }
+      let(:sp_name) { 'Your friendly Government Agency' }
+
+      it 'does not show the service provider name if not signed up via a service provider' do
+        sign_in_and_2fa_user
+        loa3_sp_session
```

Comment:
Hmm. I tried simulating what `loa3_sp_session` without an `:sp` and the flow wasn't working... 🤔